etc/config-template.toml

```diff
@@ -1047,12 +1047,6 @@
 ## If you set this configuration item to `0`, compression is disabled.
 # batch-compression-threshold = "8KB"
 
-## Specifies the maximum accumulative size of buffered writes. When this
```

Comment:
What's the new sync behavior?

Reply:
There's no limit of buffered writes now.

Reply:
Will it cause too many large IO?

Reply:
It will if you write a lot with sync=false, but it shouldn't be a problem with tikv. Note that the IO of one write group will not be split in both the old and new version.